gatsby-node.ts

```diff
@@ -41,6 +41,12 @@ export const onCreateNode: GatsbyNode['onCreateNode'] = ({ node, actions }) => {
 
 export const createPages: GatsbyNode['createPages'] = ({ actions }) => {
   const { createRedirect } = actions;
+  createRedirect({
+    fromPath: '/verify/e-ids/swedish-bankid/#bankid-phone-authentication',
```

Comment:
That won't work, it will redirect for the entire swedish-bankid page :) Servers don't see the #fragment

Reply:
Oh no! No way to create a redirect then?